e2e/test/scenarios/filters-reproductions/filters-reproductions.cy.spec.js

```diff
@@ -269,7 +269,7 @@ describe("issue 20683", { tags: "@external" }, () => {
     popover().within(() => {
       cy.findByText("Created At").click();
       cy.findByText("Relative dates…").click();
-      cy.findByText("Previous").click();
+      cy.findByText("Past").click();
```

Comment:
Reverts this
https://github.com/metabase/metabase/pull/44299/files#diff-b8ac0b262f21ddaf28d49d07c061d8919f0013c6980022cafd995aff5b43aa77R272